simulateddata/mocks.js

```diff
@@ -0,0 +1,239 @@
+const barCharData = [
```

Comment:
I think it makes more sense to put the title of the folder as "mocks" and inside it an index.js file or you can keep the current name but use a naming convention (camelCase, snake_case, kebak-case) to make it easier to read, which he thinks?